### ISSUE TITLE
Counter-Strike 1.6: Missed all fonts

### ISSUE THREAD

**Issue by ghost** [ValveSoftware/halflife#2071]
When i run Counter-Strike 1.6 on my Linux Mint 14 i do not see any text in game, even i can not see what i am writing in console.

My config:
Processor Information:
    Vendor:  GenuineIntel
    Speed: 3001 Mhz
    4 logical processors
    4 physical processors
    HyperThreading:  Unsupported
    FCMOV:  Supported
    SSE2:  Supported
    SSE3:  Supported
    SSSE3:  Supported
    SSE4a:  Unsupported
    SSE41:  Supported
    SSE42:  Supported

Network Information:
    Network Speed:  

Operating System Version:
    Linux Mint 14 Nadia (64 bit)
    Kernel Name:  Linux
    Kernel Version:  3.5.0-17-generic
    X Server Vendor:  The X.Org Foundation
    X Server Release:  11300000
    X Window Manager:  Marco
    Steam Runtime Version:  <Runtime enabled>

Video Card:
    Driver:  NVIDIA Corporation GeForce GTX 560/PCIe/SSE2

```
Driver Version:  1.4 (2.1.2 NVIDIA 310.19)
Desktop Color Depth: 24 bits per pixel
Monitor Refresh Rate: 60 Hz
VendorID:  0x10de
DeviceID:  0x1201
Number of Monitors:  1
Number of Logical Video Cards:  1
Primary Display Resolution:  1920 x 1080
Desktop Resolution: 1920 x 1080
Primary Display Size: 20.08" x 11.30"  (23.03" diag)
                                        51.0cm x 28.7cm  (58.5cm diag)
Primary Bus: PCI Express 16x
Primary VRAM Not Detected
Supported MSAA Modes:  2x 4x 8x 16x 
```

Sound card:
    Audio device: Realtek ALC887-VD

Memory:
    RAM:  7969 Mb

Miscellaneous:
    UI Language:  English
    LANG:  en_US.UTF-8
    Microphone:  Not set
    Total Hard Disk Space Available:  112675 Mb
    Largest Free Hard Disk Block:  94713 Mb

**Comment by calexil** [comment]
resolved?

**Comment by sim590** [comment]
I am getting the same issue here. Here is screenshot of what I'm getting:

![cs](https://user-images.githubusercontent.com/4147254/82292931-b67d9f00-9979-11ea-8535-8d1a34214fb2.png)

In order to go around it, I have to go (blindly) through the interface to find the options for changing the resolution and this restarts the game and texture and text now don't disappear.

However, once the game is restarded, I can't join any server as the server lists are all empty (see #2895, or [my comment on this issue][2895-comm]). As I'm explaining in this comment, I can sort of go around the issue by joinging a server blindly with the interface that has text gone as the interface is still functional. However, I never know what game I'm joining. I normally order them by ping (blindly) and select the first one.

However, if I once hit the keys `shift+tab`, then the bug is triggered again and all textures and text disappear. When in game, it shows like this:

![texture](https://user-images.githubusercontent.com/4147254/82294843-c5198580-997c-11ea-8182-e561846e4a58.png)

So, in summary, I can still "play" if I'm cautious with the keys I'm hitting and if I don't care the server I'm joining and if I don't mind having to restart the game several times (changing the resolution), but that's quite annoying.

[2895-comm]: https://github.com/ValveSoftware/halflife/issues/2895#issuecomment-630614882

**System info**

* **OS**: Archlinux
* **Archlinux' Steam package version**: 1.0.0.61-4
* **Counter Strike version**: I don't know...

**Comment by shawns-valve** [comment]
Closing due to age. Please reopen / create a new issue if this still occurs with the 25th anniversary update (December 2023 Steam build) on a current Linux distribution.
